sdk/opendp/whitenoise/client/__init__.py

```diff
@@ -34,6 +35,17 @@ def submit(self, params, uri):
         return client.executerun(details)
 
 class DatasetClient(object):
+    def readreleased(self, dataset_name, budget):
+        client = _get_client()
+        read_released_request = DatasetReadRequest(dataset_name=dataset_name, budget=budget)
+        return client.datasetreadreleased(read_released_request)
+
+    def release(self, dataset_name, budget=None):
+        client = _get_client()
+        release_request = DatasetReleaseRequest(dataset_name=dataset_name, \
```

Comment:
keeping this on one line is fine. We can add a tox.ini at the top level directory to specify the flake8 settings. We can use 100 characters as our max line length for now, this will for starters hopefully avoid your editor auto adding \. But over time it will allow us to reintroduce the flake8 build

Reply:
https://flake8.pycqa.org/en/latest/user/configuration.html

Reply:
will add flake in future PR